modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -344,6 +344,41 @@ Added 1 constraint.
 
 ======
 
+.Create `VECTOR` property type constraints label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2025.xx]
```

Comment:
you had one todo about show output but I also have a comment regarding show constraints. In the output column information for show constraints we should update 
```
| createStatement
| Statement used to create the constraint.
| STRING
```
to be
```
| createStatement
| Statement used to create the constraint, or `null` if the constraint cannot be created through `CREATE CONSTRAINT`.
| STRING
```
or something similar. This to cover that it should be null in Cypher 5 for property type constraints on vectors as well as for dependent constraints once graph types are released (so I want it more generic now as to not need to update it twice)